.python-version

```diff
@@ -0,0 +1 @@
+3.10.8
```

Comment:
probably don't want to commit this file to the repo

Reply:
whoops! should have removed that one!